test/AppNotificationTests/AppNotifications.Test.h

```diff
@@ -93,7 +95,8 @@ namespace AppNotificationHelpers {
             return;
         }
 
-        VERIFY_IS_FALSE(!expected || !actual);
+        VERIFY_IS_TRUE(!!expected);
+        VERIFY_IS_TRUE(!!actual);
```

Comment:
Nit: `VERIFY_IS_NOT_NULL`, so you clearly state the intention of these verification steps, instead of casting.